HW2_nekrasova/calculator.py

```diff
@@ -0,0 +1,62 @@
+def main(expression: str):
+    expression_array = expression.split(' ')
+    array_of_operands = []
+
+    try:
+        left_int = int(expression_array[0])
+        array_of_operands.append(left_int)
+    except:
+       left_float = float(expression_array[0])
+       array_of_operands.append(left_float)
```

Comment:
```suggestion
        left_float = float(expression_array[0])
        array_of_operands.append(left_float)
```

Куда потерялся пробел?😢